daisy_workflows/image_import/inflate_file.wf.json

```diff
@@ -86,7 +86,8 @@
           "networkInterfaces": [
             {
               "network": "${import_network}",
-              "subnetwork": "${import_subnet}"
+              "subnetwork": "${import_subnet}",
+              "accessConfigs": []
```

Comment:
I don't see why inflater would need an external IP so this is probably OK. We should just keep an eye on any unforseen issues. If there are non, we should do the same for the exporter.

Reply:
sgtm. I did a few test runs and everything looked  good.